modules/logging-loki-zone-aware-rep.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="logging-loki-zone-aware-rep_{context}"]
+= Zone aware data replication
+
+In the {logging} 5.8 and later versions, the Loki Operator offers support for zone-aware data replication through pod topology spread constraints. Enabling this feature enhances reliability and safeguards against log loss in the event of a single zone failure. When configuring the deployment size as `1x.small` or `1x.medium,` the `replication.factor` field is automatically set to 2. However, replication is not included in the `1x.extra-small` size.
```

Comment:
```suggestion
In the {logging} 5.8 and later versions, the Loki Operator offers support for zone-aware data replication through pod topology spread constraints. Enabling this feature enhances reliability and safeguards against log loss in the event of a single zone failure. When configuring the deployment size as `1x.small` or `1x.medium,` the `replication.factor` field is automatically set to 2.
```

I suggest to remove that last mention about the `1x.extra-small` as per 5.8.0 we productized this deployment size with a default replication factor of `2`.

Reply:
Would make more sense to add it to the sentence prior then. I don't want to say 'all sizes' in case we add some later.